src/cpp11.cpp

```diff
@@ -479,10 +479,6 @@ extern SEXP R_igraph_write_graph_lgl(SEXP, SEXP, SEXP, SEXP, SEXP);
 extern SEXP R_igraph_write_graph_ncol(SEXP, SEXP, SEXP, SEXP);
 extern SEXP R_igraph_write_graph_pajek(SEXP, SEXP);
 extern SEXP UUID_gen(SEXP);
-extern SEXP make_lazy(SEXP, SEXP, SEXP);
```

Comment:
this file is read-only, how was I supposed to remove these lines then? :sweat_smile:

Reply:
Regenerate using `R -e 'cpp11::cpp_register()'`. This is why I keep insisting that these procedures must be documented in wiki pages.

Reply:
oh right, thanks! or the contributing guide, a more usual place for R packages. :smile_cat: